ts/features/itwallet/common/utils/__tests__/itwClaimsUtils.test.ts

```diff
@@ -143,3 +145,36 @@ describe("extractFiscalCode", () => {
     expect(extractFiscalCode("RANDOM_STRING_MRARS001H1B")).toEqual(O.none);
   });
 });
+
+describe("ImageClaim", () => {
+  const base64 =
+    "iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAIAAACQd1PeAAAAEElEQVR4nGKKrzEGBAAA//8CVAERMRFlewAAAABJRU5ErkJggg==";
+  it("should decode a valid png image", () => {
+    const decoded = ImageClaim.decode(`data:image/png;base64,${base64}`);
+    expect(E.isRight(decoded)).toBe(true);
+  });
+  it("should decode a valid jpg image", () => {
+    const decoded = ImageClaim.decode("data:image/jpg;base64,${base64}");
```

Comment:
Shouldn't you use backticks to interpolate `${base64}`?

Reply:
There was a commit I did not push, sorry.